recipes-soletta/soletta/soletta_git.bb

```diff
@@ -47,6 +47,7 @@ FILES_${PN}-dev = " \
 FILES_${PN} = " \
             ${bindir}/sol* \
             ${libdir}/libsoletta.so* \
+            ${libdir}/node_modules/soletta \
```

Comment:
please add a split-package `PACKAGES = ... ${PN}-nodejs`, then you add these to `FILES_${PN}-nodejs`. Your nodejs-runtime in Ostro will then depend on it. Since the bindings depend on soletta, it will pull everything automatically.